lib/node.py

```diff
@@ -95,7 +95,7 @@ def ready_node(self):
 
     def docker_ps(self, all=False):
         result = self.execute_command(
-            'docker ps --format "{{.Names}}\t{{.Image}}"')
+            'docker ps -a --format "{{.Names}}\t{{.Image}}"')
```

Comment:
@soumyalj Why do we want to look for docker process is stopped stated as well ? This method could be used in places to check strictly for docker process that are in running state.

Reply:
This is because service-sidekick container shows up only in `docker ps -a`.
Modified code to handle this case.